packages/fxa-settings/src/pages/Signup/Confirm/mocks.tsx

```diff
@@ -2,4 +2,6 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-export const MOCK_EMAIL = 'bleep@bloop.com';
+export const MOCK_GOBACK_CB = () => {
```

Comment:
This seems like a strange file name change, `LinkRememberPassword` mocks to `Signup/Confirm` mocks?

Reply:
LinkRememberPassword no longer needs mocks, so it might have been caused by copying/moving the old file.